packages/dashboard/src/components/iot-dashboard/iot-context-menu/iot-context-menu-option.tsx

```diff
@@ -0,0 +1,27 @@
+import { Component, h, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'iot-context-menu-option',
+  styleUrl: 'iot-context-menu-option.css',
+  shadow: false,
+})
+export class IotContextMenuOption {
+  @Prop() disabled: boolean;
+  @Prop() onClick?: (event: MouseEvent) => void;
+
+  render() {
+    const handleClick = (event: MouseEvent) => {
```

Comment:
should be a method on the class rather than creating a new handler on every render

Reply:
👍